src/wallet/wallet.cpp

```diff
@@ -1572,7 +1572,7 @@ void CWalletTx::GetAmounts(std::list<COutputEntry>& listReceived,
  * @return Earliest timestamp that could be successfully scanned from. Timestamp
  * returned will be higher than startTime if relevant blocks could not be read.
```

Comment:
Now that this method doesn't return a timestamp (`int64_t`) anymore, this Doxygen comment needs to be updated.

Reply:
Updated, thanks!